docs/development/adding_tests.md

```diff
@@ -1,18 +1,28 @@
 # Adding tests to TheRock
 
-### Test Flow
+## Test Flow
 
 After TheRock builds its artifacts, we test those artifacts through [`test_packages.yml`](../../.github/workflows/test_packages.yml). The testing flow works as:
 
-`test_sanity_check` -> `configure_test_matrix` -> `test_components`
+`test_sanity_check`--> `configure_test_matrix` --> `test_components`
 
 where we:
 
 1. Check that the artifacts pass sanity tests.
 1. The `configure_test_matrix` step runs [`fetch_test_configurations.py`](../../build_tools/github_actions/fetch_test_configurations.py), where we generate a test matrix for which tests to run.
 1. After we generate the matrix, `test_components` executes those tests in parallel.
 
-### Adding tests
+### How these tests are executed
+
+These tests are retrieved from [`fetch_test_configurations.py`](../../build_tools/github_actions/fetch_test_configurations.py), where we generate a matrix of tests to run for various AMD GPU families from [`amdgpu_family_matrix.py`](../../build_tools/github_actions/amdgpu_family_matrix.py) on both Linux and Windows test machines.
+
+These tests are run per pull request, main branch commit, `workflow_dispatch` and nightly runs.
+
+### What kind of tests are suitable for TheRock
+
+Since `TheRock` is the open source build system for `HIP` and `ROCm`, tests that prove `HIP` and `ROCm` functionality are the tests we are looking for! Particuarly, any tests that prove component functionality or overall `ROCm`/`HIP` functionality
```

Comment:
Drop backticks from project names. Then I would also rephrase to highlight that we are most interested in testing combined functionality of subprojects, especially where there are dependency chains and the whole assembly needs to build and run correctly.

```suggestion
Since TheRock is the open source build system for HIP and ROCm, we are interested in tests for individual subprojects as well as tests that exercise multiple subprojects, especially where there are build and runtime dependencies. We also perform higher level testing of overall user-facing behavior and downstream frameworks like PyTorch.
```